lib/embed/embed.dart

```diff
@@ -281,12 +284,12 @@ class PlaygroundMobile {
     _clearOutput();
   }
 
-  void _export() {
+  void _export(WindowBase exportWindow) {
     PadSaveObject exportObject = new PadSaveObject()..html = context.htmlSource
```

Comment:
I suspect we can move the open call to inside this method - it'll just have to be before the Future's `then()` call.